WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -87,84 +97,121 @@ function DelegatesOfRegion({ activeSeniorDelegate, delegates, isAdminMode }) {
   );
 }
 
-export default function Delegates({
-  delegates,
-  isEditVisible,
-}) {
-  const seniorDelegates = React.useMemo(() => delegates
+export default function Delegates() {
+  const { data, loading, error } = useLoadedData(delegatesStaticPageDataUrl);
+
+  const { delegates, canViewDelegateMatters } = data || {};
+  const seniorDelegates = React.useMemo(() => !!delegates && delegates
     .filter((user) => user.delegate_status === 'senior_delegate')
     .sort((user1, user2) => (user1.location || '').localeCompare(user2.location || '')), [delegates]);
 
   const [activeSeniorDelegate, setActiveSeniorDelegate] = React.useState(seniorDelegates?.[0]);
```

Comment:
Since `seniorDelegates` will be falsey on initial render now, and to avoid duplicating this logic from the use effect.
```suggestion
  const [activeSeniorDelegate, setActiveSeniorDelegate] = React.useState();
```